docarray/utils/create_dynamic_doc_class.py

```diff
@@ -0,0 +1,236 @@
+from docarray import DocList, BaseDoc
+from docarray.typing import AnyTensor
+from pydantic import create_model
+from typing import Dict, List, Any, Union, Optional, Type
+
+
+def create_new_model_cast_doclist_to_list(model: Any) -> BaseDoc:
```

Comment:
the name is a bit confusing

Reply:
but does it need to be a public method ? If not they lets just make it private and no need to change the name I guess

Reply:
it needs to be yes

Reply:
Okay I see, well maybe the name is good them,  I cannot find a better one

Reply:
or maybe `create_pure_python_type_model` ? Who knows maybe when we will introduce DocDIct we will have to convert DOcDict to Dict as well in this method and therefore the name with DocList is not relevant anymore

Reply:
okey

Reply:
done